.github/workflows/analyze.yml

```diff
@@ -36,8 +37,17 @@ jobs:
         if: ${{ inputs.sdk == 'dart' }}
       - uses: subosito/flutter-action@48cafc24713cca54bbe03cdc3a423187d413aafa # pin@v2.10.0
         if: ${{ inputs.sdk == 'flutter' }}
+
       - run: ${{ inputs.sdk }} pub get
-      - run: ${{ inputs.sdk }} format --set-exit-if-changed ./
+
+      - run: dart format .
+
+      - run: dart fix --apply
```

Comment:
Should we still do `format --set-exit-if-changed ./` after this just for the sake of validity?

Reply:
`dart fix` works but I am not sure if for 100% of the cases.

Reply:
`dart fix` does not affect formatting AFAIK, it's just used for things linke unused imports and such, which gets detected by `dart analyze`. This i did leave in for exactly this reason, issues that `dart fix` cannot resolve. Formatting fixes will always work with just running `dart format .`